spec/integration/api/v1/config_spec.rb

```diff
@@ -32,6 +32,8 @@ def app()
     let(:current_time) { Time.now }
 
     before(:each) do
+      allow(Vmpooler::API).to receive(:run!)
+      Vmpooler::API.execute(['api'], :config, :redis, :metrics)
```

Comment:
What is this was `Vmpooler::API.execute(['api'], config, redis, metrics)`, given the `lets` defined above? It seems like the `app.settings.set` would be made redundant if that worked.

Reply:
Additionally, you can override/extend lets from the parent contexts.

Reply:
Done - I've left in the override statements.